### ISSUE TITLE
chef enables rabbitmq_management_visualiser in all runs

### ISSUE THREAD

**Issue by maneeshmp** [rabbitmq/chef-cookbook#411]
Operating system: Ubuntu 14.04.2 
chef-client version: 12.15.19

Attributes (default.rb)

``` ruby
 default['rabbitmq']['enabled_plugins'] = ['rabbitmq_management', ' rabbitmq_management_visualiser']
```

recipe(rbmq.rb)

``` ruby
   include_recipe 'rabbitmq::plugin_management'
```

Chef-shell

chef (12.14.60)> cmd.stdout
```
 => " Configured: E = explicitly enabled; e = implicitly enabled
 | Status:   * = running on rabbit@abc-dev-01-app01\n |/
[E*] rabbitmq_management_visualiser 3.6.1" 
chef (12.14.60)> cmd.stdout =~ /#{name}/
 => 125 
```

chef-debug log

```
recipe: rabbitmq::plugin_management
  * rabbitmq_plugin[rabbitmq_management] action enable (up to date)
  * rabbitmq_plugin[ rabbitmq_management_visualiser] action enable
    * execute[rabbitmq-plugins enable  rabbitmq_management_visualiser] action run
      [execute] Plugin configuration unchanged.
                
                Applying plugin configuration to rabbit@abc-dev-01-app01... nothing to do.
      - execute rabbitmq-plugins enable  rabbitmq_management_visualiser
....................
..................
{kernel,{net_ticktime,60}}]
 * Restarting message broker rabbitmq-server
   ...done.

    - restart service service[rabbitmq-server]
```

I believe something is going wrong with the method

``` ruby
def plugin_enabled?(name)
  ENV['PATH'] = "#{ENV['PATH']}:/usr/lib/rabbitmq/bin"
  cmdstr = "rabbitmq-plugins list -e '#{name}\\b'"
  cmd = Mixlib::ShellOut.new(cmdstr, :env => shell_environment)
  cmd.run_command
  Chef::Log.debug "rabbitmq_plugin_enabled?: #{cmdstr}"
  Chef::Log.debug "rabbitmq_plugin_enabled?: #{cmd.stdout}"
  cmd.error!
  cmd.stdout =~ /\b#{name}\b/
end
```

Even with debug run I don't see the,   "Chef::Log.debug" log in the output, so may be cmd.run_command returning false ?

**Comment by michaelklishin** [comment]
@jjasghar `rabbitmq_management_visualiser` is deprecated and will not ship with RabbitMQ 3.7.0. If it's on the default plugin list I think we should drop it. WDYT?

**Comment by michaelklishin** [comment]
Addressed in master/`5.5.0`.
